tests/Fixer/Phpdoc/PhpdocTypesOrderFixerTest.php

```diff
@@ -138,9 +138,19 @@ public static function provideFixWithNullFirstCases(): iterable
                 '<?php /** @var Foo[]|null|Foo|Foo\Bar|Foo_Bar */',
             ],
             [
-                '<?php /** @method void bar(null|string $delimiter = \',<br/>\') */',
+                '<?php /** @method null|Y|X setOrder(array<string, null|array{Y,X,null|Z}> $by) */',
+                '<?php /** @method Y|null|X setOrder(array<string, array{Y,X,Z|null}|null> $by) */',
+            ],
+            '@method with invalid phpdoc - simple' => [
+                '<?php /** @method null|X setOrder($$by) */',
+                '<?php /** @method X|null setOrder($$by) */',
+            ],
+            '@method with invalid phpdoc - default value is not supported' => [
                 '<?php /** @method void bar(string|null $delimiter = \',<br/>\') */',
             ],
+            '@method with invalid phpdoc - with generic' => [
+                '<?php /** @method static setOrder(array<string, string> &&$by) */',
+            ],
```

Comment:
I believe this shouldn't be part of `provideFixWithNullFirstCases()` provider, because:

- this is non-fix case
- it does not relate to "null first"

Reply:
done, `tests\Fixtures\Integration\misc\phpdocs-invalid.test` test is enought and stronger